compiler/damlc/lib/DA/Cli/Options.hs

```diff
@@ -176,6 +176,11 @@ dotFileOpt = option (Just <$> str) $
     <> long "dot"
     <> value Nothing
 
+htmlOutFile :: Parser FilePath
+htmlOutFile = argument str $
```

Comment:
I prefer using `-o outfile` for output arguments over having multiple positional arguments. It’s much easier if you don’t have to remember the order.

Reply:
I agree with @cocreature here, `-o output` is a bit nicer.